app/routes/content.js

```diff
@@ -104,7 +104,8 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
                 ['og:description', node.get('description')],
                 ['og:site_name', this.get('theme.provider.name')],
                 ['og:type', 'article'],
-                ['article:published_time', new Date(preprint.get('dateCreated') || null).toISOString()]
+                ['article:published_time', new Date(preprint.get('datePublished')).toISOString()],
```

Comment:
An API bug? Interesting....

Since this deals with a known edge case, it's ok to leave in and approve for move forward as written.

As long as the API behavior is tracked and ticketed, we're good here. Thanks for verifying. A fix is outside the scope of this PR and we can wait until the dust settles.